.azure-pipelines.yml

```diff
@@ -203,4 +203,4 @@ stages:
       displayName: 'test'
       env:
         AZURE_ACCESS_TOKEN: "$(System.AccessToken)"
-        TFLAGS: "-u -vc /usr/bin/curl.exe -r -rm $(tests)"
+        TFLAGS: "-r -rm -u !SCP $(tests)"
```

Comment:
Does anyone ever read the `-r` generated output? I know I never do it.

Reply:
I do sometimes as one way to figure out potential reasons for test flakiness and timeouts.